src/components/data/ScrollBox.tsx

```diff
@@ -6,6 +6,8 @@ import ScrollContent from './ScrollContent';
 export type ScrollBoxProps = CompositeProps & {
   /** Turn off borders on outer container */
   borderless?: boolean;
+  /** Add rounded corners to scrollable container */
+  rounded?: boolean;
```

Comment:
It's hard to imagine the different scenarios in which we use scrollable containers, so I'm adding rounded corners as an optional opt-in feature

Reply:
To confirm, the current default of `false` matches the existing look?

Reply:
Yes. It will continue being non-rounded (current behavior) unless actively opting in.